lib/api/controllers/securityController.js

```diff
@@ -251,9 +255,83 @@ class SecurityController extends BaseController {
       'updateUserMapping',
       'validateCredentials'
     ]);
+
     this.subdomain = 'security';
+    // @todo Modify all security action with default "refresh" to "wait_for"
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)